handlers.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type HandlersEnv struct {
```

Comment:
I have a rename suggestion
```suggestion
type DeviceHandlers struct {
```